webapp/src/components/header/settings-dialog/SettingsDialog.tsx

```diff
@@ -93,6 +93,19 @@ export const SettingsDialog: React.FC<ISettingsDialogProps> = ({ open, closeDial
                                     })}
                                 </AccordionPanel>
                             </AccordionItem>
+                            <Divider />
+                            <AccordionItem value="about">
+                                <AccordionHeader expandIconPosition="end">
+                                    <h3>About</h3>
+                                </AccordionHeader>
+                                <AccordionPanel>
+                                    <Body1 color={tokens.colorNeutralForeground3}>
+                                        Frontend version: {process.env.REACT_APP_SK_VERSION ?? "-"}<br/>
+                                        {process.env.REACT_APP_SK_BUILD_INFO}<br/>
+                                        Backend version: {serviceOptions.version}
+                                    </Body1>
+                                </AccordionPanel>
+                            </AccordionItem>
```

Comment:
nit: add a divider after so there's separation with the footer when scrolled all the way down
```suggestion
                            </AccordionItem>
                            <Divider />
```
<img width="537" alt="Screenshot 2023-08-15 at 10 06 31 AM" src="https://github.com/microsoft/chat-copilot/assets/125500434/8e35d12d-2492-4145-aa27-b1bb764234f3">